### PR TITLE
fix: deployments -> set rights only on group and others, not on user

### PR DESCRIPTION
**Pull request by MichaelBitard**
## Why is the change needed
This is an annoying issue.

## Description of the change
We don't try to recursively set the user rights anymore, only the group (piaf-deployment) and others.

### CODE REVIEW

File: deployment/roles/haystack/tasks/main.yml [hunk 1]
```diff
@@ -77,3 +77,10 @@
   community.docker.docker_compose:
     project_src: "{{ client_installation_directory }}/"
     build: yes
+- name: "{{ client }} | change group ownership (https://github.com/ansible/ansible/issues/51601)"
```
Comment (guillim) [inline deployment/roles/haystack/tasks/main.yml:80]: maybe it's a stupid question, but why is this line necessary since we already have a group ownership defined line 27 ?

Comment (MichaelBitard) [reply deployment/roles/haystack/tasks/main.yml:80]: Not stupid at all, this is an issue in ansible, related to the git module: https://github.com/ansible/ansible/issues/51601

Comment (guillim) [reply deployment/roles/haystack/tasks/main.yml:80]: so we need to specify it twice. ok

**Review by guillim — APPROVED** [review]
